gems/smithy/spec/fixtures/all_shapes/model.smithy

```diff
@@ -76,4 +76,7 @@ map ComplexMap {
 union Union {
     string: String,
```

Comment:
You can remove integer here, I just needed two values. I think we also want simple list and simple map (for documentation tests?)